ui/analyse/src/study/studyTour.ts

```diff
@@ -23,3 +23,9 @@ export function chapter(setTab: (tab: string) => void) {
     });
   });
 }
+
+export function cancelCurrentTour() {
+  lichess.loadScript('javascripts/study/tour-chapter.js').then(() => {
+    window.lichess['cancelCurrentTour']();
```

Comment:
You can just do normal dot access `window.lichess.cancelCurrentTour()`, no?

Reply:
Hmm admittedly I didn't try it with normal dot access and it seems like it should work -- is there value in keeping it consistent style-wise with [other similar functions in this file that don't use dot access](https://github.com/lichess-org/lila/blob/a8374127321b88e9278b79344f983e075a6718bd/ui/analyse/src/study/studyTour.ts#L21)?

Reply:
I'd rather adjust the other functions. This is slightly odd style and shouldn't be necessary.